File: app/src/main/java/com/market/solarcharger/PortResponseInfo.java
package com.market.solarcharger;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class PortResponseInfo {

    @SerializedName("error")
    @Expose
    private Boolean error;
    @SerializedName("port")
    @Expose
    private List<portDetailInfo> portDetailInfoList;

    public Boolean getError() {
        return error;
    }

    public void setError(Boolean error) {
        this.error = error;
    }

    public List<portDetailInfo> getPortDetailInfoList() {
        return portDetailInfoList;
    }

    public void setPortDetailInfoList(List<portDetailInfo> portDetailInfoList) {
        this.portDetailInfoList = portDetailInfoList;
    }

}

File: app/src/main/java/com/market/solarcharger/PortResponseInfo.kt
package com.market.solarcharger

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class PortResponseInfo {
    @SerializedName("error")
    @Expose
    var error: Boolean? = null

    @SerializedName("port")
    @Expose
    var portDetailInfoList: List<portDetailInfo>? = null

}